src/foam/lib/parse/CSVParser.java

```diff
@@ -66,7 +66,7 @@ public PStream parse(PStream ps, ParserContext px) {
             }
             sink.put(obj, detach);
           } catch ( Exception e ) {
-            return null;
+           // return null;
```

Comment:
Don't just comment this out. Leave a comment as to why you want to do nothing when a value fails to parse and remove the `return null`

Reply:
Added a comment.

Reply:
Done.